pkg/operator/ceph/csi/version.go

```diff
@@ -112,3 +116,29 @@ func extractCephCSIVersion(src string) (*CephCSIVersion, error) {
 
 	return &CephCSIVersion{major, minor, bugfix}, nil
 }
+
+// SupportsCustomCephConf checks if the detected version supports custom ceph.conf
+func (v *CephCSIVersion) SupportsCustomCephConf() bool {
+	// if AllowUnsupported is set also a csi-image greater than the supported ones are allowed
+	if AllowUnsupported {
```

Comment:
Why do we need this check? Can we wait for this code to merge after we use ceph-csi 3.5.0? So we don't need this?

Reply:
This is to support the custom cephcsi who are building cephcsi with a different versions (ex:- OCS).